backend/lobby/consumers.py

```diff
@@ -79,15 +81,21 @@ def get_or_create_user(self, username, room):
 
     def get_or_create_game(self, username, room):
         game, created = GameState.objects.get_or_create(room_name=room)
-
         i_repository_game = IRepositoryGame()
+        i_repository_player = IRepositoryPlayer()
 
         if not i_repository_game.get_game_by_room(room):
             game_store = i_repository_game.save_game(room)
             print("Room created with id {}".format(game_store))
         else:
             error = 'Room already exist, Unable to create Game with room: ' + room
```

Comment:
See comment below, I think these error messages could be a little misleading. Approving but you may want to consider adjusting them to let us know gameplay is working but analytics are not

Reply:
updated this error as well message to clarify